src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,211 @@
+package duke;
+import java.io.*;
+import java.util.Scanner;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+import duke.exception.EmptyDescriptionException;
+import duke.exception.InvalidCommandException;
+import java.util.ArrayList;
+
+import duke.utilityfunctions.Filereader;
+public class Duke {
+    public static void generateTaskStatus(String taskIcon, String statusIcon, String description) {
+        System.out.println("\t[" + taskIcon + "]" + "[" + statusIcon + "] " + description);
+    }
+    public static void drawLine() {
```

Comment:
Consider adding at least 1 line space between the methods to improve readability